src/features/reddit-feed/index.jsx

```diff
@@ -2,12 +2,25 @@ import React from 'react';
 import { NewsfeedPanel } from '../../components/NewsfeedPanel';
 import { Newsfeed } from './Newsfeed';
 import useAppState from 'state/AppStateProvider';
+import { useStyles } from './styles';
 
 function RedditFeed() {
   const { openLiveStream } = useAppState();
+  const styles = useStyles();
+
+  const title = () => {
+    return (
+      <div className={styles.root}>
```

Comment:
@joshslee is there any way we can factor this all out into a separate component? There's a similar one in the `TwitterFeed`

Reply:
@jchen006 sounds good.